packages/nux/src/components/guide/README.md

```diff
@@ -0,0 +1,56 @@
+Guide
```

Comment:
I wonder if this could be just a `@wordpress/components` component instead

Reply:
👌 moved `Guide` over to `@wordpress/components` in 78104f3db0190d6fac295933d44bee00a5d15c9f.